pkg/reconciler/service/kube/kube.go

```diff
@@ -0,0 +1,205 @@
+/*
+Copyright 2020 The Knative Authors
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package kube
```

Comment:
Is this only useful for broker? If so, can you move this under the pkg/reconciler/broker/ file structure?

The naming and location is also non-standard for knative, it is not reconciling services, it is a sub-component that realizes requests for some kind of addressable deployment. Perhaps move to `pkg/reconciler/internal/deploymentservice` ?

Reply:
I like the "internal" one. I kept `kube` path because I think it's kinda align with `kube` in several packages in `pkg` where resources are k8s native.